azure-pipelines.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       sudo apt-get install apt-transport-https
       sudo apt-get update
       sudo apt-get install dotnet-sdk-2.2
-      ./build.sh --target Test --configuration Release
+      ./build.sh --target=Test --configuration=Release
```

Comment:
Apparently Cake cares that you use the format `--param=value` and no longer accepts `--param value`. We may need to change this in other projects if we update Cake.